src/app/core/services/auth.service.ts

```diff
@@ -74,30 +73,68 @@ export class AuthService {
     return this.authStateSource.getValue() === AuthState.Authenticated;
   }
 
-  setLoginStatusWithGithub(isLoggedIn: boolean) {
-    if (!isLoggedIn) {
-      this.electronService.clearCookies();
-    } else {
-      this.electronService.setCookie(this.githubUrl, this.githubDomain, this.githubLoginCacheName, 'yes');
-    }
-  }
-
   changeAuthState(newAuthState: AuthState) {
     if (newAuthState === AuthState.Authenticated) {
       const sessionId = `${Date.now()}-${uuid()}`;
       this.issueService.setSessionId(sessionId);
-      Logger.info(`Successfully authenticated with session: ${sessionId}`);
+      this.logger.info(`Successfully authenticated with session: ${sessionId}`);
     }
     this.authStateSource.next(newAuthState);
   }
 
   /**
    * Will start the Github OAuth web flow process.
-   * @param clearAuthState - A boolean to define whether to clear any auth cookies so prevent auto login.
    */
-  startOAuthProcess(clearAuthState: boolean = false) {
+  startOAuthProcess() {
     const githubRepoPermission = this.phaseService.githubRepoPermissionLevel();
     this.changeAuthState(AuthState.AwaitingAuthentication);
-    this.electronService.ipcRenderer.send('github-oauth', clearAuthState, githubRepoPermission);
+
+    if (this.electronService.isElectron()) {
+      this.electronService.sendIpcMessage('github-oauth', githubRepoPermission);
+    } else {
+      const authService = this;
+      const oauthWindow = this.createOauthWindow(encodeURI(
+        `${AppConfig.githubUrl}/login/oauth/authorize?client_id=${AppConfig.clientId}&scope=${githubRepoPermission},read:user`
+      ));
+      oauthWindow.addEventListener('unload', () => {
```

Comment:
Suggestion: let's move the event handler registration into `createOauthWindow` as well.
`startOAuthProcess` will become less detailed.